### ISSUE TITLE
Runs with additional-app-test-apks fail

### ISSUE THREAD

**Issue by inktomi** [Flank/flank#775]
We use the `additional-app-test-apks` array to test our library submodules. At the end of that run, the job always fails when we use the latest Flank release, with the following exception. It seems like there's an issue in how the results are being processed? 

```bash
FetchArtifacts
  ......................................................................................................................................................
  Updating matrix file

java.lang.IllegalStateException: testCase.startTime must not be null
	at ftl.reports.api.CreateTestExecutionDataKt.getStartTimestamp(CreateTestExecutionData.kt:52)
	at ftl.reports.api.CreateTestExecutionDataKt.createTestExecutionData(CreateTestExecutionData.kt:39)
	at ftl.reports.api.CreateTestExecutionDataKt$createTestExecutionData$1.invokeSuspend(CreateTestExecutionData.kt)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.ResumeModeKt.resumeUninterceptedMode(ResumeMode.kt:45)
	at kotlinx.coroutines.internal.ScopeCoroutine.afterCompletionInternal(Scopes.kt:32)
	at kotlinx.coroutines.JobSupport.completeStateFinalization(JobSupport.kt:310)
	at kotlinx.coroutines.JobSupport.tryFinalizeFinishingState(JobSupport.kt:236)
	at kotlinx.coroutines.JobSupport.tryMakeCompletingSlowPath(JobSupport.kt:849)
	at kotlinx.coroutines.JobSupport.tryMakeCompleting(JobSupport.kt:811)
	at kotlinx.coroutines.JobSupport.makeCompletingOnce$kotlinx_coroutines_core(JobSupport.kt:787)
	at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:111)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:241)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:594)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.access$runSafely(CoroutineScheduler.kt:60)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:740)
```

Here's the `flank.yml` that we use to run the tests

```yaml
flank:
  additional-app-test-apks:
  - test: feature/corp-pay/build/outputs/apk/androidTest/debug/corp-pay-debug-androidTest.apk
  - test: feature/login/build/outputs/apk/androidTest/debug/login-debug-androidTest.apk
....(many more -test:)....
  local-result-dir: sub-modules-flank-results
  max-test-shards: 5
  project: firebase-project-100015
  smart-flank-gcs-path: gs://firebase-project-100015-flank/m1-submodules-timing.xml
gcloud:
  app: ./sub-modules-tester/build/outputs/apk/debug/sub-modules-tester-debug.apk
  async: false
  auto-google-login: false
  device:
  - model: NexusLowRes
    version: 23
  environment-variables:
    clearPackageData: true
  performance-metrics: false
  record-video: false
  results-bucket: sub-modules-flank-results
  test: fake-permissions/build/outputs/apk/androidTest/debug/fake-permissions-debug-androidTest.apk
  timeout: 7m
  use-orchestrator: true
```

**Comment by bootstraponline** [comment]
Thanks for reporting!

**Comment by bootstraponline** [comment]
Thanks for reporting! This issue has been fixed and released in Flank v20.05.1.
